src/js/lib/admin-page/style.scss

```diff
@@ -0,0 +1,165 @@
+$gray: #777;
+
+#wp-parsely-react-entrypoint {
+	.components-snackbar {
+		border-left: 5px solid green;
+		&.settings-error {
+			border-left: 5px solid red;
+		}
+	}
+	#wp-parsely_version {
+		color: $gray;
+		font-size: 12px;
+		margin-left: 1em;
+	}
+
+	.help-text {
+		width: 75%;
+	}
+
+	.controls {
+		display: flex;
+		flex-direction: row;
+	}
+
+	.nav-control {
+		cursor: pointer;
+		justify-content: space-between;
+		height: 2vh;
+		font-size: larger;
+		padding: 1em;
+	}
+
+	.inactive {
+		display: none;
+	}
+
+	span.active {
+		font-weight: bolder;
+		text-decoration: underline;
+	}
+
+	.setting-item {
+		display: inline-flex;
+		flex-direction: row;
+		justify-content: space-between;
+		width: 50%;
+	}
+
+	.setting-item--control {
+		width: 100%;
+		max-width: 300px;
+	}
+
+	.site-detail {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+		max-width: 300px;
+		padding-bottom: 1em;
+	}
+
+	.details-info {
+		display: flex;
+		flex-direction: column;
+		border: 1px solid #ccc;
+		width: 100%;
+		padding: 5% 20% 5% 5%;
+	}
+
+	.details-label {
+		width: 50%;
+		font-size: large;
+		padding-bottom: 1em;
+	}
+
+	.code {
+		font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+	}
```

Comment:
Switch to `<code>` element, and let the browser or wp-admin default handle the styling of this.